src/components/NavigationComponent/NavigationComponent.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import { connect } from "react-redux";
+import { actionsToggle } from "../../actions";
```

Comment:
I don't see this chain of functions being used anywhere and the component works fine, so I think it can be removed.

Reply:
removed both actionsToggle and connect since its not being used